index.js

```diff
@@ -463,14 +484,15 @@ class GithubScm extends Scm {
             return repo.data.permissions;
         } catch (err) {
             // Suspended user
-            if (err.code === 403) {
+            if (err.code === 404) {
```

Comment:
ref.: https://developer.github.com/v3/troubleshooting/
> Typically, we send a 404 error when your client isn't properly authenticated. You might expect to see a 403 Forbidden in these cases. However, since we don't want to provide any information about private repositories, the API returns a 404 error instead.

Reply:
It can get 404 and fail already in the previous function call `lookupScmUri`.

Reply:
Ah, it can also get 404 in `lookupScmUri` when the repository does not exist any more. Can we somehow check the err message from github and make sure then it's the case of account suspended then do whatevet inside the block?
```
                 winston.info(
                     `User's account suspended for ${scmInfo.owner}/${scmInfo.repo}, ` +	                     `User's account suspended for ${scmInfo.owner}/${scmInfo.repo}, ` +
                     'it will be removed from pipeline admins.');	                     'it will be removed from pipeline admins.');
 	 
                 return { admin: false, push: false, pull: false };	                 return { admin: false, push: false, pull: false };
```

Reply:
Sorry i didn't make it clear in the first comment. Didn't really think through that at that time :(

Reply:
I want to avoid to check the error message, because its message may be modified in the future.
But status code 404 for suspended accounts seems to be ambiguous too.
I think we have no choice but to check the error message.

Reply:
I fixed it.